kafka-bom/build.gradle.kts

```diff
@@ -0,0 +1,23 @@
+plugins {
```

Comment:
Sorry I had this yesterday and forgot to push it. This will allow downstreams to align on the same version, and inherit all dependency constraints.